server/routes/user.js

```diff
@@ -0,0 +1,25 @@
+const router = require('express').Router();
+const User = require('../database/models/user');
+const Order = require('../database/models/order');
+
+router.get('/', (req, res, next) => {
+  User.findAll()
+    .then(users => res.json(users))
+    .catch((e) => {
+      console.error('error in all users route', e);
+      next(e);
+    });
+});
+
+router.get('/:id/orders', async (req, res, next) => {
```

Comment:
This looks good, I think we could exclude status: 'Cart'

Reply:
fixed this. went with the easy way but added functionality so we could create a dropdown for users to see statuses if we have time